addons/mail/static/src/js/composers/basic_composer.js

```diff
@@ -32,6 +32,8 @@ var BasicComposer = Widget.extend({
         'keydown .o_composer_input textarea': '_onKeydown',
         'keyup .o_composer_input': '_onKeyup',
         'click .o_composer_button_send': '_sendMessage',
+        'drop .o_file_drop_zone_container': '_onFileDragDropped',
+        'paste .o_composer_input': '_onPasteImage',
```

Comment:
remove copy-paste feature. It is not working, and after discussing with EST we should drop this feature from this task